src/test/resources/application.conf

```diff
@@ -0,0 +1,36 @@
+// a test DB is required.  Here's an example command to run one locally
```

Comment:
Hm.  Skimming https://github.com/typesafehub/config it seems the distinction is app.conf for applications and ref.conf for libraries - so I think app conf is right